### ISSUE TITLE
Readiness check

### ISSUE THREAD

**Issue by augier** [Root-Branch/cardamon-core#78]
Enjoying playing around with Cardamon!

Would it be worth adding an option for a readiness check similar to is implemented in k8s?

Having your service simply return a 200 vs other code to state that it is ready to receive traffic and is in it's "settled" state.

This could be an optional configuration item on the process.

**Comment by ohuu** [comment]
Yeah that sounds like a good idea. The only problem I can think of is that (if I've understood you correctly) it would require some code changes to the program being measured. As long as its something that's opt-in and we fall back to the simple method then I'm all for it.

**Comment by ohuu** [comment]
We could have another line in the config called health_check which provides a list of containers which cardamon performs a health_check on rather than assuming docker containers are good to go after a specified delay. Health check is a part of docker (check this [out](https://docs.docker.com/reference/dockerfile/#healthcheck))

health_check = [<container_name>, ...]
